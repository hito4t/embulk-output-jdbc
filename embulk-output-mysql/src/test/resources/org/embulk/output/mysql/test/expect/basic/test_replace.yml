table: test1
mode: replace
