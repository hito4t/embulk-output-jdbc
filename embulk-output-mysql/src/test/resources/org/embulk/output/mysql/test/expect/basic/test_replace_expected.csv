1,123.40,test1,TEST1,2015-04-23 16:02:03
2,1234567890.12,test9999,TEST9999,2015-12-31 15:00:00
